cmd/broker/main_test.go

```diff
@@ -99,25 +98,137 @@ var _ = Describe("Blockhead", func() {
 				}
 			})
 
-			It("should successfully return services", func() {
-				catalogURL := fmt.Sprintf("%s%s", serverAddress, "/v2/catalog")
+			var requestCatalog = func() *http.Response {
+				catalogURL := fmt.Sprintf("%s/v2/%s", serverAddress, "catalog")
 				req, err = http.NewRequest("GET", catalogURL, nil)
 				Expect(err).NotTo(HaveOccurred())
 				req.SetBasicAuth("test", "test")
 				req.Header.Add("X-Broker-API-Version", "2.0")
 
-				Eventually(poller, 5*time.Second).Should(Succeed())
-				Expect(resp.StatusCode).To(Equal(http.StatusOK))
+				var resp *http.Response
+				Eventually(func() error {
+					resp, err = client.Do(req)
+					return err
+				}).Should(Succeed())
+				return resp
+			}
 
+			var requestBind = func(serviceId, planId, instanceId string) *http.Response {
```

Comment:
this is the wrong name, it's for provision, right?

Reply:
yup, that should be `requestProvision` :disappointed: